src/solvers/dg.jl

```diff
@@ -172,6 +172,43 @@ function Base.show(io::IO, ::MIME"text/plain", integral::VolumeIntegralPureLGLFi
 end
 
 
+"""
+    VolumeIntegralSubcellLimiting(indicator;
+                                  volume_flux_dg, volume_flux_fv)
+
+A subcell limiting volume integral type for DG methods based on subcell blending approaches
+with a low-order FV method. Used with indicator [`IndicatorIDP`](@ref).
+
+!!! warning "Experimental implementation"
+    This is an experimental feature and may change in future releases.
+"""
```

Comment:
Reference?

Reply:
I've added the references to the specific indicators, since the concept behind both approaches is quite different.